server.js

```diff
@@ -49,6 +53,19 @@ function authenticate(code, cb) {
   req.on('error', function(e) { cb(e.message); });
 }
 
+function log(label, value, sanitized) {
+  value = value || '';
+  if (sanitized){
+    if (typeof(value) === 'string' && value.length > 10){
```

Comment:
Replace the hard-coded `10` with a named constant to avoid introducing “magic numbers”.